source/security/firmware-update.rst

```diff
@@ -1,37 +1,372 @@
 Firmware Update
 ------------------
 
-|SPN| implements a robust, secure and power fail-safe firmware update mechanism. |SPN| image can be built to contain redundant boot components and depending on hardware assisted boot partition switch, to support fail-safe update.
+|SPN| implements a secure and power fail-safe firmware update mechanism.
 
-If firmware update is interrupted by a power failure and one boot partition is corrupted, |SPN| boots from the alternate partition to avoid bricking the platform. The firmware update authenticates the update image (a.k.a. capsule) before updating the firmware.
+  * |SPN| image is designed to contain redundant boot components and depends on hardware assisted boot partition switch to support fail-safe update.
 

```

Comment:
Add a couple of points here
* Firmware update code is implemented as a payload
* |SPN| detects firmwate update signal and launches the Firmware Update payload

Reply:
Done